src/magellan_core/launch/line_detector.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+        <node pkg="magellan_vision" type="line_detector.py" name="line_detector_node"/>
+        <param name="VERBOSE" type="boolean" value="false"/>
```

Comment:
can we change this to lower case like everything else?

Reply:
fixed